README.md

```diff
@@ -1 +1,28 @@
 # Weather-App
+
+This project is one of the homeworks for the Code Your Future coding training course. This application is created to let user check the temperature of the city of their desire. It takes users input and fetch data accordingly from the server and displays the fetched data in the browser in a way that is easy to read to the users.
+
+
+# Team members
+
+#Prabesh
```

Comment:
if you use `##` for our names this will create a better structure https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
ok I will follow your advice and pull request again